src/service/viewport-impl.js

```diff
@@ -1068,7 +1072,8 @@ export class ViewportBindingNaturalIosEmbed_ {
     if (!this.scrollMoveEl_) {
       return;
     }
-    setStyle(this.scrollMoveEl_, 'transform', `translateY(${scrollPos}px)`);
+    setStyle(this.scrollMoveEl_, 'transform',
+        `translateY(${scrollPos - this.paddingTop_}px)`);
```

Comment:
Could you please clarify re:"double calculated"? When `transform` is set, the padding is removed and when scroll position is calculated, it's added back.